codeface_extraction/codeface_extraction.py

```diff
@@ -107,11 +109,12 @@ def run():
     # - First make all the args absolute
     __resdir = abspath(args.resdir)
     __codeface_conf, __project_conf = map(abspath, (args.config, args.project))
+    __exclude_impl = args.implementation
```

Comment:
`!args.implementation`, I guess. If the `--implementation` parameter is given, you do **not** want to _exclude_ the function implementations. Otherwise, the name of the variable would be misleading.